tests/integrationtest/t/executor/infoschema_reader.test

```diff
@@ -130,6 +130,8 @@ connection default;
 create table tt (a int);
 alter table tt add index t(a), add column b int;
 select db_name, table_name, job_type from information_schema.DDL_JOBS limit 3;
+alter table tt add index t2(a), add unique index tb(b), add column c int;
+select db_name, table_name, job_type from information_schema.DDL_JOBS limit 4;
```

Comment:
why not limit 7 to show items for later DDL, else i am not sure what's this part is checking

Reply:
I want to check there's only one add index sub-job, so it's really merged. `limit 4` can show the "multi-schema" line of last DDL so we know all lines belongs to this DDL is printed

Reply:
there is a `alter table tt add index t(a), add column b int;` above, the result is the similar😓

Reply:
this PR tests about merging unique and non-unique add index. I can delete that case in next commit

Reply:
https://github.com/pingcap/tidb/pull/53632/commits/d46188912fbf94940e33ccfb165090122d30ba7c